plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -90,9 +92,25 @@ global proc mayaUSDRegisterStrings() {
     register("kSaveOptionNoPromptAnn", "You can re-enable this prompt under File|Save Scene Options.");
     register("kSaveOptionTitle", "Save USD Options");
     register("kSaveOptionUnsavedEdits", "You have unsaved USD edits. How would you like to proceed?");
+    register("kUSDSelectionMode", "USD Selection Mode");
+    register("kUSDSelectionModeAnn", "Assign selection categories to prims via the Attribute Editor to activate selection modes in this menu. Choose a selection mode to reflect changes when you select prims in the Viewport and Outliner. Note: By default, the fallback selection mode is by prim. If a selection mode is applicable, it is applied to your scene but otherwise defaults to selecting by prim.");
+    register("kUSDSelectionModeAssembly", "Assembly");
+    register("kUSDSelectionModeAssemblyAnn", "Selection mode for prims set to assembly kind. Tip: Set assembly kind in the Attribute Editor to prims that are part of an important group.");
+    register("kUSDSelectionModeComponent", "Component");
+    register("kUSDSelectionModeComponentAnn", "Selection mode for prims set to component kind. Tip: Set component kind in the Attribute Editor to prims that are a collection of assets.");
+    register("kUSDSelectionModeCustom", "Selection mode for prims set to ^1s kind.");
+    register("kUSDSelectionModeGroup", "Group");
+    register("kUSDSelectionModeGroupAnn", "Selection mode for prims set to group kind. This includes prims set to assembly kind. Tip: Set group kind in the Attribute Editor to prims that are grouped.");
+    register("kUSDSelectionModeModel", "Model");
+    register("kUSDSelectionModeModelAnn", "Selection mode for prims in the model hierarchy. This includes prims set to group, assembly and component kind.");
+    register("kUSDSelectionModeNone", "(none)");
+    register("kUSDSelectionModeNoneAnn", "Selection mode for prims that have no kind set.");
+    register("kUSDSelectionModeSubComponent", "Subcomponent");
+    register("kUSDSelectionModeSubComponentAnn", "Selection mode for prims set to subcomponent kind. Tip: Set subcomponent kind in the Attribute Editor to prims that are an individual asset.");
     register("kTimeAnn", "Edits the current time value of a stage, which corresponds to the animation frame drawn in the viewport. By default, this value connects to Maya's global time node.");
     register("kTipYouCanChooseMultipleFiles", "<b>Tip:</b>  You can choose multiple files.");
     register("kUnloadAll", "Unload All");
+    register("kUniversalSceneDescription", "Universal Scene Description");
```

Comment:
This string already exists with "(USD)" at the end and is used where we add the stages menu. See tag kMenuStageSubMenu above. They are probably supposed to be the same. Please check with Natalia to see if they should be the same and if so we don't need this extra tag (just use one above).

Reply:
I spoke with Natalia and she will think about it. So I think for now that should be fine